src/classes/class/Class.js

```diff
@@ -0,0 +1,35 @@
+import {useMemo} from "react";
+import {useParams} from "react-router-dom";
+import CLASSES from "../example";
+import PersonCard from "./PersonCard";
+import useUsers from "../../hooks/useUsers";
+import H1 from "../../components/H1";
+
+function Class() {
+  const {classURL} = useParams();
+
+  // if CLASSES was large, then this would be expensive so useMemo would be useful
+  const classData = useMemo(() => {
+    return CLASSES.find(c => c.url === classURL)
+  }, [classURL]);
+
+  const [users, message] = useUsers();
+
+  return (
+    <div className="px-8 py-10">
+      <H1>{classData.title}</H1>
+      <p className="mt-3 text-md text-neutral-gray">{classData.description}</p>
+      <p className="mt-4 text-lg text-neutral-gray">{classData.members} members</p>
+      <div className="mt-2 flex flex-nowrap gap-3 overflow-x-auto">
+        {users.length === 0 ?
+          <p className="mt-4 test-md text-neutral-gray">{message}</p> :
+          users.map((user) => (
+            <PersonCard {...user} key={user.id}/>
+          ))
+        }
+      </div>
+    </div>
```

Comment:
I recommend making more child components to make your code more readable and clean up this component a bit more.
For high-level components with lots of children, especially if the children have different purposes/focuses, hiding away implementation details in other components is really important and useful.
For example, your `Class` component can be summarized by "a title, a description, a member count, and list of users".
your return statement would thus look like:
```js
return (
  <div>
    <Title/>
    <Description />
    <MemberCount />
    <UsersList />
  </div>
)
```
This way, it's a lot cleaner and easier for someone to get a rough idea of what's inside this component. If they then wanted more details about the `UsersList`, they could find the `UsersList` component defined in `UsersList.js` and get all the details they need.
In this way, your `Class` component acts more like a table of contents than the contents themselves.